codesystems/CodeSystem-England-PrescriptionType.xml

```diff
@@ -0,0 +1,559 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+   <id value="England-PrescriptionType" />
+   <url value="https://fhir.nhs.uk/CodeSystem/England-PrescriptionType" />
+   <version value="1.0.0" />
+   <name value="EnglandPrescriptionType" />
+    <title value="England PrescriptionType" />
```

Comment:
PrescriptionType -> Prescription Type. Also bring start of tag inline with the top and bottom tags.